pages/tech-per-quiz.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { Button } from "../components/ui/Button";
+
+const Intake = () => {
+    const [onBoarding, setOnBoarding] = useState({
+        name: "",
+        email: ""
+    });
+
+    const handleChange = (e) => {
+        const { name, value } = e.target
+        setOnBoarding((prev) => {
+            return { ...prev, [name]: value };
+        });
+    };
+
+    const handleSubmit = (e) => {
+        // e.preventDefault();
+        console.log(onBoarding);
+    };
```

Comment:
Great work, do you mind moving the handle functions to their own component, keep this in the same folder